.parcelrc

```diff
@@ -1,19 +1,25 @@
 {
   "extends": "@parcel/config-default",
-  "resolvers": ["parcel-resolver-docs", "..."],
+  "resolvers": ["@parcel/resolver-glob", "parcel-resolver-docs", "..."],
+  "bundler": "@parcel/bundler-experimental",
```

Comment:
This fixes the issue with some examples not rendering which we had patched Parcel before, disabling optimization. The new bundler is currently in progress and experimental, but it seems to build our docs correctly.